.golangci.yml

```
@@ -0,0 +1,153 @@
+linters-settings:
+  errcheck:
+    # report about not checking of errors in type assertions: `a := b.(MyStruct)`;
+    # default is false: such cases aren't reported by default.
+    check-type-assertions: true
+  goconst:
+    # minimal occurrences count to trigger, 3 by default
+    min-occurrences: 5
+  gocritic:
+    # Enable multiple checks by tags, run `GL_DEBUG=gocritic golangci-lint run` to see all tags and checks.
+    # Empty list by default. See https://github.com/go-critic/go-critic#usage -> section "Tags".
+    enabled-tags:
+      - diagnostic
+      - experimental
+      - opinionated
+      - performance
+      - style
+    disabled-checks:
+      - whyNoLint
+      - emptyStringTest
+  goimports:
+    # put imports beginning with prefix after 3rd-party packages;
+    # it's a comma-separated list of prefixes
+    local-prefixes: github.com/platformsh/platformify
```

```suggestion
    local-prefixes: github.com/platformsh/cli
```
after this change, it is necessary to regroup imports in several files, otherwise the liner will fail